meinberlin/apps/budgeting/assets/ControlBar.jsx

```diff
@@ -1,88 +1,66 @@
-import React, { useState, useEffect } from 'react'
+import React, { useState } from 'react'
 import { ControlBarDropdown } from './ControlBarDropdown'
 import { ControlBarListMapSwitch } from './ControlBarListMapSwitch'
 import { ControlBarSearch } from './ControlBarSearch'
 import { ControlBarSearchTerm } from './ControlBarSearchTerm'
 import { SpacedSpan } from './SpacedSpan'
 import django from 'django'
+import { useSearchParams } from 'react-router-dom'
 
 const translated = {
   showFilters: django.gettext('Show filters'),
   hideFilters: django.gettext('Hide filters'),
-  filters: django.gettext('Filters'),
-  results: django.gettext('results found.')
+  filters: django.gettext('Filters')
+}
+
+const getResultCountText = (count) => {
+  const foundProposalsText =
+    django.ngettext('1 proposal found.', '%s proposals found.', count)
+  return django.interpolate(foundProposalsText, [count])
```

Comment:
Not really sure what interpolate does here, it reads like it's mostly if there is more than one number?

Reply:
tbh i just copied from `BudgetProposalList`, tried without but doesn't seem to work without `interpolate()`.

Reply:
ah, it's different in js than in python! cool, works nicely